docs/development/local-development.md

```diff
@@ -49,15 +49,15 @@ If you already installed a component, skip the corresponding step.
 
   Verify you're using the appropriate versions:
```

Comment:
I don't like that we're telling people *exact* versions to use when a range would actually suffice (this PR doesn't make this worse.. but worth discussing)

Reply:
How about something like this then?
This sounds more like "here's how you check your versions" instead of "you must use these versions".

```suggestion
  You can see what versions you're using like this:
```

Reply:
We do have a minimum version required in many cases though. So I'd prefer to specify the minimum version and then the commands which can be used to check it

Reply:
Do you want me to remove the version output from the PowerShell commands then?

That way we can rely on the versions listed at the top of the document (L13 trough L18).

Reply:
Good idea

Reply:
I've removed the version output.
Did you want to use the suggestion I made above to make the text less "you must use this version or else!!"?

Reply:
Yes, please do